CMakeLists.txt

```diff
@@ -342,18 +395,21 @@ target_link_libraries(ufs_model PRIVATE ufs
 ###############################################################################
 ### Install
 ###############################################################################
-install(
-  TARGETS fms
-  EXPORT fms-config
-  LIBRARY DESTINATION lib
-  ARCHIVE DESTINATION lib
-  PUBLIC_HEADER DESTINATION include )
+if(FMS)
+  install(
```

Comment:
This block will be removed in https://github.com/ufs-community/ufs-weather-model/pull/372

Reply:
addressed in 3624511